static/js/services/vulnerability-service.js

```diff
@@ -127,7 +127,9 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
           'introducedInVersion': feature.Version,
 
           'imageId': addedByImageId,
```

Comment:
We should rename this variable and maybe even the `imageId` field here

Reply:
Will do in follow up.